pkg/gardenlet/bootstrap/util/util.go

```diff
@@ -138,6 +138,10 @@ func UpdateGardenKubeconfigCAIfChanged(ctx context.Context, seedClient client.Cl
 		return kubeconfig, nil
 	}
 
+	if bytes.Equal(gardenClientConnection.GardenClusterCACert, []byte("none")) || bytes.Equal(gardenClientConnection.GardenClusterCACert, []byte("null")) {
```

Comment:
🥴 What about simply checking for `gardenClientConnection.GardenClusterCACert=nil`? Why do we need magic strings like `none` or `null`?

Reply:
I could do that, but then it will be a breaking change, since the field is not required in the helm values right now and therefore probably defaults to `nil`, if not set.

Reply:
Okay, fair point. Can you add some documentation about this to https://github.com/gardener/gardener/blob/master/docs/deployment/deploy_gardenlet_manually.md?

Reply:
Done.